homeassistant/components/teslemetry/coordinator.py

```diff
@@ -15,6 +17,9 @@
 from homeassistant.exceptions import ConfigEntryAuthFailed
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
+if TYPE_CHECKING:
+    from . import TeslemetryConfigEntry
+
```

Comment:
May be move the Entry type to `coordinator.py`?

Reply:
I left the config entry type there, since the config entry runtime data does not only contain the coordinator. Not sure if this consideration is correct, but i'm open to discuss it :)